python_modules/dagster/dagster/_check/builder.py

```diff
@@ -260,6 +260,8 @@ def build_check_call_str(
     name: str,
     eval_ctx: EvalContext,
 ) -> str:
+    from dagster._record import _RECORD_MARKER_FIELD
```

Comment:
can just use  `is_record`  if we keep this

Reply:
Updated 👍